src/chunk_store/tests.rs

```diff
@@ -90,13 +88,9 @@ async fn successful_put() -> Result<()> {
     let chunks = Chunks::gen(&mut rng);
 
     let root = temp_dir();
-    let used_space = Arc::new(Mutex::new(0));
-    let mut chunk_store = unwrap!(ChunkStore::<Data>::new(
-        root.path(),
-        u64::MAX,
-        Arc::clone(&used_space),
-        Init::New
-    ));
+    let used_space = UsedSpace::new(u64::MAX);
+    let mut chunk_store =
+        unwrap!(ChunkStore::<Data>::new(root.path(), used_space.clone(), Init::New).await);
```

Comment:
@Scorch-Dev we're currently striving to cleanup tests, to return a `Result` type. Would be great to use that here instead of `unwrap`.
```suggestion
        ChunkStore::<Data>::new(root.path(), used_space.clone(), Init::New).await?;
```